### PR TITLE
modify implementations.json

### PR DESCRIPTION
**Pull request by nuoyimanaituling**
this is a sample implementation of raft ，and can help some students like me to understand raft

### CODE REVIEW

File: implementations.json [hunk 1]
```diff
@@ -2250,5 +2250,23 @@
...
             "persistence": true
         }
+    },
+    {
+        "repoURL": "https://github.com/nuoyimanaituling/xzwraft",
+        "name": "xzwRaft",
+        "authors": [
+            {
+                "name": "xzw",
+                "github": "xzw"
```
Comment (ongardie) [inline implementations.json:2260]: Did you intend this to be @xzw or @nuoyimanaituling ?

File: implementations.json [hunk 2]
```diff
@@ -2250,5 +2250,23 @@
...
+        "license": "Apache-2.0",
+        "features": {
+            "basic": true,
+            "membershipChanges": false,
+            "logCompaction": false,
+            "persistence": true
+        }
     }
-]
+]
```
Comment (ongardie) [inline implementations.json:2272]: Please restore this newline to satisfy `check.py`

File: implementations.json (hunk 1 above)
Comment (nuoyimanaituling) [reply implementations.json:2260]: oh，sorry，i forget it，i intend this to be @nuoyimanaituling

**Comment by nuoyimanaituling** [comment]
hi，Diego Ongaro，I am not very familiar with git，so i may make a mistake to pull request, and i have pulled a new commit
hope you can receive, very thanks!

**Comment by ongardie** [comment]
Thanks @nuoyimanaituling. You did exactly the right thing by adding another commit and pushing to the same branch.

Now it looks like `check.py` is still failing with:

> Implementations should be sorted by repoURL

Do you mind fixing that? That's the end of the script, so I think that must be the last error.

**Comment by nuoyimanaituling** [comment]
Hi，Diego Ongaro，thanks for your prompt reply. Is it because I put it at the end?  i think my order is not very important to me.If necessary，i will update it.

**Comment by ongardie** [comment]
Yes, the `check.py` script requires the implementations to be sorted by repoURL. This helps keep the file organized, and it helps avoid merge conflicts when multiple people submit pull requests concurrently.

Later, another script (`fetch_repo_info.py`) that runs on my web server determines the ordering that's displayed on the website.

**Comment by nuoyimanaituling** [comment]
So that's it，thanks your reply，and what should i do next？

**Comment by ongardie** [comment]
@nuoyimanaituling I went ahead and moved it to be in sorted order. All done now. Thanks!

**Pull request merged**